net-im/skypeforlinux/skypeforlinux-9999.ebuild

```diff
@@ -9,7 +9,7 @@ inherit eutils rpm
 
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
-SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
+SRC_URI="https://repo.skype.com/latest/skypeforlinux-64-alpha.rpm"
```

Comment:
This really ain't going to work. Are you able to determine some kind of version from the file?

You may also want to see how other distros do this.